docs/_partners/common-migration-errors.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Common Errors When Migrating Users
+order: 11
+layout: post-toc
+redirect_from: /partners/
+---
+<a id="deploy-errors"></a>
+
+## Common Errors When Migrating Users and Zaps Between Versions
+
+There are certain changes to an integration the platform considers breaking changes, meaning migrating users and their Zaps to the new version will break existing, live Zaps and cause them to be turned off. 
+
+Zapier's platform runs basic automated checks to prevent situations that may break Zaps before migrating users. But the checks can't cover everything, so please be extra aware of possible breaking changes when updating your integrations. No checks exist for version promotions, as having existing incompatible versions in parallel is acceptable.
+
+Here are some commonly encountered errors and breaking changes to watch out for, with suggested solutions:
+
+> **Note:** You can also always choose not to migrate users, and follow our [deprecation process](https://platform.zapier.com/cli_tutorials/versions#deprecate-an-older-version-of-your-integration).
+
+### You cannot remove or update keys for existing Triggers, Actions, and Searches.
+
+You will see a similar error to the one below from our automated checks:
+`vX.X is missing the following keys: {<list of missing keys>}. Hide them instead of removing them.`
+
+The Triggers, Actions, and Searches are identified by their **key**, so if you remove it, or change its **key**, this message appears.
+
+There are two solutions:
+
+- If you need to remove a Trigger/Action/Search, change its visibility to **hidden** instead.
```

Comment:
Might link to how you adjust visibility (or just mention that there's a specific key to set)